.travis.yml

```diff
@@ -60,6 +60,12 @@ matrix:
         - CMAKE_OPTIONS="-DRModular=ON -DTRAVIS_DISABLE_UNIT_TESTS=ON -DTRAVIS_DISABLE_LIBSHOGUN_TESTS=ON"
         - CC=clang
         - CXX=clang++
+    - compiler: clang
+      services: docker
+      env:
+        - CMAKE_OPTIONS="-DScalaModular=ON -DTRAVIS_DISABLE_UNIT_TESTS=ON -DTRAVIS_DISABLE_LIBSHOGUN_TESTS=ON"
+        - CC=clang
+        - CXX=clang++
```

Comment:
which needs a sep PR, I suggest do modify docker config first (it auto-updates the image when we merge a PR to the config file), and then this travis change in a single PR.
Then the scala stuff you made in a next PR